lib/Menu.rb

```diff
@@ -0,0 +1,45 @@
+class Menu
+  attr_reader :meal_options, :dishes, :customer_total, :store_order
+
+  MEAL_OPTIONS = {:Hamburger=>5, :Pizza=>6, :Pasta=>4,:Fish_and_chips=>5,:Salad=>3,:Lamb_Shank=>7}
+
+    def initialize(printer = Print.new, bill = BillCalculator.new(MEAL_OPTIONS))
+      @dishes = dishes
```

Comment:
What is the `dishes` on the right hand side of this line of code?

To get some visibility on what is happening, try creating a new Menu instance in `irb` and seeing what the value of `dishes` is